plugins.json

```diff
@@ -946,5 +946,17 @@
         "download": "https://github.com/ipunkt/KanboardTaskButtonsReposition/releases/download/1.0.0/KanboardTaskButtonsReposition-1.0.0.zip",
         "remote_install": true,
         "compatible_version": ">=1.2.5"
+    },
+    "ProjectInvitation": {
+        "title": "Project Invitation",
+        "version": "1.0.0",
+        "author": "Andrei Volgin / Hussein Khalil, ipunkt Business Solutions",
+        "license": "MIT",
+        "description": "Plugin is used to invite new users to actual Project by typing user's email. Input for invitation is located in Project Settings->Permissions.",
+        "homepage": "https://github.com/ipunkt/KanboardProjectInvitation",
+        "readme": "https://github.com/ipunkt/KanboardProjectInvitation/blob/master/README.md",
+        "download": "https://github.com/ipunkt/KanboardProjectInvitation/releases/download/v1.0.0/KanboardProjectInvitation-1.0.0.zip",
```

Comment:
The folder name must match the plugin namespace.

```bash
$ unzip KanboardProjectInvitation-1.0.0.zip 
Archive:  KanboardProjectInvitation-1.0.0.zip
56a19c7ea67e67874e16ce9e59f099f8f498045e
   creating: KanboardProjectInvitation/
   creating: KanboardProjectInvitation/Controller/
  inflating: KanboardProjectInvitation/Controller/ProjectInviteController.php  
   creating: KanboardProjectInvitation/Data/
  inflating: KanboardProjectInvitation/Data/config.php  
  inflating: KanboardProjectInvitation/LICENSE  
   creating: KanboardProjectInvitation/Locale/
   creating: KanboardProjectInvitation/Locale/de_DE/
  inflating: KanboardProjectInvitation/Locale/de_DE/translations.php  
   creating: KanboardProjectInvitation/Locale/ru_RU/
  inflating: KanboardProjectInvitation/Locale/ru_RU/translations.php  
  inflating: KanboardProjectInvitation/Plugin.php  
  inflating: KanboardProjectInvitation/README.md  
   creating: KanboardProjectInvitation/Template/
  inflating: KanboardProjectInvitation/Template/users-override.php  
  inflating: KanboardProjectInvitation/Template/users.php  
   creating: KanboardProjectInvitation/docker/
  inflating: KanboardProjectInvitation/docker-compose.yml  
   creating: KanboardProjectInvitation/docker/conf.d/
  inflating: KanboardProjectInvitation/docker/conf.d/local.ini
```

```php
<?php

namespace Kanboard\Plugin\ProjectInvitation;

use Kanboard\Core\Plugin\Base;
```

Reply:
@fguillot We have changed the namespace in Plugin. Everything in website request stays the same.

`<?php

namespace Kanboard\Plugin\KanboardProjectInvitation;

use Kanboard\Core\Plugin\Base;`

Reply:
@fguillot We have changed the `Namespace` in our Plugin as well as in Website.

`namespace Kanboard\Plugin\KanboardProjectInvitation;`